### ISSUE TITLE
VIVO Error: User cancelled image selection

### ISSUE THREAD

**Issue by yoyo0427** [ivpusic/react-native-image-crop-picker#1816]
### Version
Tell us which versions you are using: 

- react-native-image-crop-picker v0.23.1
- react-native v0.59.5

### Platform
Tell us to which platform this issue is related

- Android
- VIVO V2057A ; vivo ROM:Funtouch OS_10.5;  Android10

### Expected behaviour

Without opening the album, the information in the catch is  Error: User cancelled image selection

It turned out to be possible, but the problem occurred after the mobile phone system was upgraded.

### Actual behaviour

Ability to open albums and select photos

### Steps to reproduce

1.

2.

3.

### Attachments

// stacktrace or any other useful debug info

![图片](https://user-images.githubusercontent.com/12933263/179901932-f49f0b7f-e35f-443e-b4a5-367962a0110d.png)

Love react-native-image-crop-picker? Please consider supporting our collective:
👉  https://opencollective.com/react-native-image-crop-picker/donate

**Comment by Aliveing** [comment]
+1

**Comment by kurisu994** [comment]
Change PickerModule.java
```java
final Intent chooserIntent = Intent.createChooser(galleryIntent, "Pick an image");
activity.startActivityForResult(chooserIntent, IMAGE_PICKER_REQUEST);
```
to

```java
activity.startActivityForResult(galleryIntent, IMAGE_PICKER_REQUEST);
```
it worked

**Comment by Aliveing** [comment]
@kurisu994 Yeah, it works fine, but I don't know why it happens on Vivo Android 10 phones, I prefer not use [patch-package](https://github.com/ds300/patch-package) to hack it like you wrote in issue #1832 as a real solution, so in my issue #1813 I say it's a "temporary solution", and this only applies to vivo.

**Comment by diazbeltran** [comment]
@kurisu994 ty
